python/pyarrow/src/arrow/python/arrow_to_pandas.cc

```diff
@@ -826,6 +830,37 @@ Status ConvertListsLike(PandasOptions options, const ChunkedArray& data,
   return Status::OK();
 }
 
+template <class T>
+struct ListViewConversionType;
+
+template <>
+struct ListViewConversionType<ListViewType> {
+  using type = ListType;
+};
+
+template <>
+struct ListViewConversionType<LargeListViewType> {
+  using type = LargeListType;
+};
+
+template <typename T>
+enable_if_list_view<T, Status> ConvertListsLike(PandasOptions options,
+                                                const ChunkedArray& data,
+                                                PyObject** out_values) {
+  using ListViewArrayType = typename TypeTraits<T>::ArrayType;
+  using ConversionType = typename ListViewConversionType<T>::type;
+  using ConversionClass = typename TypeTraits<ConversionType>::ArrayType;
+  ArrayVector list_arrays;
+  for (int c = 0; c < data.num_chunks(); c++) {
+    const auto& arr = checked_cast<const ListViewArrayType&>(*data.chunk(c));
+    ARROW_ASSIGN_OR_RAISE(auto converted_array,
+                          ConversionClass::FromListView(arr, options.pool));
+    list_arrays.emplace_back(converted_array);
+  }
+  auto chunked_array = std::make_shared<ChunkedArray>(list_arrays);
+  return ConvertListsLike<ConversionType>(options, *chunked_array, out_values);
```

Comment:
Can you create an issue about making this conversion more efficient and mention it here? I could pick that later and adapt the list-conversion code to also work with list-views (that's tricky enough that I don't really know how to do it without actually writing the code myself).

Reply:
absolutely! That would be super helpful. I'll add it to the C++ ListView "remaining tasks" parent issue.